source/includes/k8s/file-transfer-protocol-k8s.rst

```diff
@@ -0,0 +1,167 @@
+.. versionadded:: Operator v5.0.7
+
+Overview
+--------
+
+You can use the SSH File Transfer Protocol (``SFTP``) to interact with the objects on a MinIO Operator tenant deployment.
+
+SFTP is defined by the Internet Engineering Task Force (IETF) as an extension of SSH 2.0.
+It allows file transfer over SSH for use with :ref:`Transport Layer Security (TLS) <minio-tls>` and virtual private network (VPN) applications.
+
+Enabling SFTP does not affect other MinIO features.
+
+
+Supported Commands
+~~~~~~~~~~~~~~~~~~
+
+When enabled, MinIO supports the following ``sftp`` operations:
+
+- ``get``
+- ``put``
+- ``ls``
+- ``mkdir``
+- ``rmdir``
+- ``delete``
+
+MinIO does not support either ``append`` or ``rename`` operations.
+
+Operator only supports the SFTP file transfer protocol.
+Other protocols, such as FTP, are not supported for accessing Tenants.
+
+
+Considerations
+--------------
+
+
+Versioning
+~~~~~~~~~~
+
+SFTP clients cannot read specific :ref:`object versions <minio-bucket-versioning>` other than the latest version.
+
+- For read operations, MinIO only returns the latest version of the requested object(s) to the sftp client.
+- For write operations, MinIO applies normal versioning behavior for matching object names.
```

Comment:
This feels a little awkward and depends on the user clicking through or knowing what normal versioning behavior is. Maybe break it up? Spitballing again

- For write operations, MinIO creates a new object version at the specified namespace.
  ``rm`` and ``rmdir`` operations specifically create ``DeleteMarker`` objects.

Reply:
Adopted with minor modification 👍🏻